src/core/microservices/microservices.module.ts

```diff
@@ -57,8 +59,15 @@ const subscriptionConfig: { provide: string; queueName: MessagingQueue }[] = [
   },
 ];
 
+const replicaUUID = randomUUID();
```

Comment:
You have used ```trackingID``` in subscription.factory.provider.ts. Seems like a better name and can be used here as well, for the sake of consistency.